blocks/api/categories.js

```diff
@@ -16,7 +16,12 @@ const categories = [
 	{ slug: 'formatting', title: __( 'Formatting' ) },
 	{ slug: 'layout', title: __( 'Layout Blocks' ) },
 	{ slug: 'widgets', title: __( 'Widgets' ) },
-	{ slug: 'embed', title: __( 'Embed' ) },
+	{ slug: 'embed-common', title: __( 'Common' ) },
```

Comment:
Probably not... Although this is just for getting feedback on the UX at this point :)